src/hist/plot.py

```diff
@@ -170,7 +170,7 @@ def plot_pull(
 
     # Computation and Fit
     values = self.values()
-    yerr = self.variances()
+    yerr = self.variances() ** 0.5
```

Comment:
As `variances` is a NumPy array 
```suggestion
    yerr = np.sqrt(self.variances())
```

```
variances(flow: bool = False) -> Union[numpy.ndarray, NoneType] method of hist.hist.Hist instance
    Returns the estimated variance of the accumulated values. The sum of squared
    weights for weighted histograms, the variance of samples for profiles, etc.
    For an unweighed histogram where kind == "COUNT", this should return the same
    as values if the histogram was not filled with weights, and None otherwise.
    If counts is equal to 1 or less, the variance in that cell is undefined if
    kind == "MEAN". This must be written <= 1, and not < 2; when this
    effective counts (weighed mean), then counts could be less than 2 but
    more than 1.
    
    If kind == "MEAN", the counts can be used to compute the error on the mean
    as sqrt(variances / counts), this works whether or not the entries are
    weighted if the weight variance was tracked by the implementation.
    
    Currently, this always returns - but in the future, it will return None
    if a weighted fill is made on a unweighed storage.
    
    :param flow: Enable flow bins. Not part of PlottableHistogram, but
    included for consistency with other methods and flexibility.
    
    :return: np.ndarray[np.float64]
```

can we use `np.sqrt` here instead?

```
$ ipython
Python 3.8.7 (default, Feb 13 2021, 01:38:22) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.21.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import numpy as np

In [2]: import timeit

In [3]: x = np.arange(100)

In [4]: %timeit np.sqrt(x)
710 ns ± 7.58 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [5]: %timeit x ** 0.5
2.7 µs ± 38.4 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
```

Reply:
That's disappointing. Yes, we can.

Reply:
I guess it will hardly matter for small arrays though, and most histograms will have under 100 entries.

Reply:
Not that this is likely to be performance critical. I'm sure the plotting part is much slower. But we should follow good practice.

Reply:
Sounds good to me. I hadn't actually done this test before today, but I had remembered seeing this post in 2019 https://twitter.com/HEPfeickert/status/1136730519226372096?s=20 about `math.pow(x, 2)` is more efficient than `x ** 2` and so did a quick check that my hunch that `np.sqrt` might also be faster than `x ** 0.5`.